package.json

```diff
@@ -41,7 +41,7 @@
     "tempy": "1.0.1"
   },
   "engines": {
-    "node": ">=18"
+    "node": ">=20.8.1"
```

Comment:
I picked this because 20.8.1 is the most recent security update

Reply:
makes sense to me. also aligns with recently restricted ranges i've seen:
* [ava](https://github.com/avajs/ava/blob/5975b602b771e0dc02382d24b65c5561bd5fc7ee/package.json#L39)
* [got](https://github.com/sindresorhus/got/releases/tag/v14.0.0)